packages/astro/src/content/types-generator.ts

```diff
@@ -41,6 +41,11 @@ type CreateContentGeneratorParams = {
 
 type EventOpts = { logLevel: 'info' | 'warn' };
 
+type EventWithOptions = {
+	event: ContentEvent;
```

Comment:
Small nit, but `event.type` will be more readable than using `event.event` everywhere
```suggestion
	type: ContentEvent;
```